.github/workflows/stale.yml

```diff
@@ -22,3 +22,27 @@ jobs:
           stale-pr-message: |
             This issue has been automatically marked as stale because it has not had recent activity. It will be
             closed if no further activity occurs. Thank you for your contributions.
+  stale-good-first-issues:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          days-before-stale: 21
+          stale-issue-message: |
+            This issue will be unassigned in a week if no further activity is seen. If you are active please provide an update on the status of the issue and if you would like to continue working on it..
+          include-only-assigned: true
+          remove-stale-when-updated: true
+          only-issue-labels: 'good first issue'  
```

Comment:
```suggestion
          only-issue-labels: 'good first issue'
```

Reply:
Good Morning @mowies, do we need a dash over here?

Reply:
Nope, no space needed at the end of the line. The label is correct this way